### ISSUE TITLE
Zend/Validate/Hostname.php uses deprecated option

### ISSUE THREAD

**Issue by moffe42** [zendframework/zf1#577]
The isValid() method on Zend_Validate_Hostname uses a deprecated option (in PHP 5.6)

```
iconv_set_encoding('internal_encoding', 'UTF-8');
```

throws a deprecation notice

```
iconv_set_encoding(): Use of iconv.internal_encoding is deprecated
```

The docs state 

```
Zend recommends the most current release of PHP for critical security and performance enhancements, and currently supports PHP 5.2.11 or later.
```

**Comment by mhujer** [comment]
It has been fixed for some time - https://github.com/zendframework/zf1/blob/master/library/Zend/Validate/Hostname.php#L1413 - are you using latest ZF1 relase?

**Comment by froschdesign** [comment]
This was fixed with version [1.12.7](https://github.com/zendframework/zf1/releases/tag/release-1.12.7).
